neqo-http3/src/lib.rs

```diff
@@ -184,6 +184,59 @@ use thiserror::Error;
 
 use crate::{features::extended_connect, priority::PriorityHandler};
 
+#[derive(Debug, Clone, Eq, PartialEq)]
+pub struct DatagramPayload {
```

Comment:
This seems generally useful.  Any thought about a different name?

Reply:
How about `Bytes` in accordance with https://docs.rs/bytes/latest/bytes/ ?

Would as well allow it to move into `neqo-common` and thus find other applications.